docs/document/content/user-manual/shardingsphere-jdbc/usage/sharding/spring-boot-starter.cn.md

```diff
@@ -8,7 +8,7 @@ weight = 3
 ```xml
 <dependency>
     <groupId>org.apache.shardingsphere</groupId>
-    <artifactId>shardingsphere-jdbc-core-spring-boot-starter</artifactId>
+    <artifactId>shardingsphere-jdbc-spring-boot-starter</artifactId>
```

Comment:
It is not an error.

https://search.maven.org/artifact/org.apache.shardingsphere/shardingsphere-jdbc-core-spring-boot-starter/5.0.0-beta/jar